openapi/components.yml

```diff
@@ -316,3 +316,79 @@ components:
           type: string
         user_id:
           type: string
+
+    PostTag:
+      type: object
+      properties:
+        value:
+          type: string
+          nullable: false
+          description: "tag string"
+          example: 'Tag string'
+        colour:
+            type: string
+            nullable: true
+            enum:
+              - 'RED'
+              - 'GREEN'
+              - 'BLUE'
+            example: 'GREEN'
+        creator_user_id:
```

Comment:
The code looks for an item in args called just `user_id` so please can the two match?
https://github.com/communitiesuk/funding-service-design-assessment-store/blob/2386ae17efe62f8639454f1345779c3214245eae/api/routes/tag_routes.py#L25

Reply:
good spot